### DISCUSSION TITLE
Github action for creating badges

### DISCUSSION THREAD

**Discussion by muchirijane** [EddieHubCommunity/support#107] [Ideas] [answered]
When a beginner starts to contribute they are given a specific badge on their first successful merge. As they continue contributing, they get different badges to encourage them to contribute more🔰 and they also get personalized emails 💌 with their GitHub profile pic.

**Comment by eddiejaoude** [comment, marked as answer]
I really like this idea 👍 , I love badges too 🤓 - they are like the **digital swag** 😎

**Reply by muchirijane** [reply]
Absolutely, It will encourage more developer to participate and get cool badges, 🙌

**Reply by eddiejaoude** [reply]
@muchirijane can you raise an issue so it is something we can action and do? Plus you will get another green square 🟩 for today 🤓

**Comment by TheShubham99** [comment]
@eddiejaoude
check this out, I've built something similar 😄  
https://dev.to/theshubham99/you-contribute-you-rock-24pa

**Reply by eddiejaoude** [reply]
Thanks @TheShubham99 , I will take a look 👍
